packages/backend/package.json

```diff
@@ -1,147 +1,148 @@
 {
```

Comment:
Is it linter or something? Can we revert it to avoid potential merge conflicts?

Reply:
What are you referring to?

Reply:
Amount of spaces in the entire file

Reply:
Ah, yes it looks like the file is using tabs, so when I generate the package lock file it always shows a large diff, since it looks like the package lock file is using spaces, not tabs. So I updated the package.json here to match the spacing used in other areas of the project (2 spaces).

Reply:
Does it work with the lint --fix flag using our current configuration?

Reply:
I'm not entirely sure I follow, but I just ran `npx lerna run lint` on the latest commit in this branch and I'm not seeing anything changed related to this package.json. Is that helpful?